src/app/dynamic/enterprise/quota-widget/style.scss

```diff
@@ -0,0 +1,99 @@
+//                Kubermatic Enterprise Read-Only License
+//                       Version 1.0 ("KERO-1.0”)
+//                   Copyright © 2022 Kubermatic GmbH
+//
+// 1. You may only view, read and display for studying purposes the source
+//    code of the software licensed under this license, and, to the extent
+//    explicitly provided under this license, the binary code.
+// 2. Any use of the software which exceeds the foregoing right, including,
+//    without limitation, its execution, compilation, copying, modification
+//    and distribution, is expressly prohibited.
+// 3. THE SOFTWARE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND,
+//    EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+//    MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+//    IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY
+//    CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+//    TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+//    SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+//
+// END OF TERMS AND CONDITIONS
+
+@use 'mixins';
+@use 'variables';
+
+.km-quota-widget {
+  .km-icon-quota {
+    margin-right: 10px;
+  }
+
+  .mat-card {
+    margin: 0;
+    padding: 0;
+
+    .km-icon-quota {
+      margin-left: 10px;
+    }
+  }
+
+  .km-quota-item {
+    padding-top: 9px;
+
+    .property-usage-bar {
+      @include mixins.size(60px, 5px, true);
+
+      border-radius: variables.$border-radius;
+    }
+
+    [value] {
+      margin: 0.6em 0;
```

Comment:
We're not using relative units anywhere in the app so please change this to `px`. This comment also applies to other `em` usages in this file.